modules/ROOT/pages/building-soap-proxy.adoc

```diff
@@ -94,11 +92,11 @@ These properties are set in the `basicauth.properties` config file and passed to
                                    basicAuthUser="${basic.auth.user}" basicAuthPassword="${basic.auth.password}"/>
 ----
 
-If no value is provided for the properties, then the WSDL retrieval is done without Basic Authentication.
+If no authentication values are provided for the properties, the WSDL is retrieved without the basic authentication.
 
-== Retrieving a WSDL using TLS
+== Retrieving a WSDL Using TLS
 
-As from version 2.1.0, the SOAP Proxy supports as an out of the box feature retrieving a WSDL from a remote location reusing the TLS context configuration set in API Manager. The TLS context used will be the one configured for the outbound configuration of the proxy.
+With version 2.1.0, the SOAP proxy is now provided as an out-of-the-box feature to retrieve a WSDL from a remote location by reusing the TLS context configuration that is set in API Manager. The TLS context configured for the outbound configuration of the proxy is used for the operation.
```

Comment:
Dazzling! This is so much better than what I had originally written. Thank you very much, Priya!

Reply:
With version 2.1.0 of what? Best guess...:
```suggestion
<what?> version 2.1.0 and later includes the SOAP proxy necessary to retrieve a WSDL from a remote location. Instead of you configuring authentication, the TLS context configured for the outbound configuration of the proxy is used to retrieve the WSDL.
```
In <what?> version 2.1.0 and later, you no longer need to configure basic authentication

Reply:
Done! Thanks!